### DISCUSSION TITLE
文石 BOOX Note Air 使用感想 | XX’s Blog

### DISCUSSION THREAD

**Discussion by xxchan** [xxchan/xxchan.github.io#19] [Announcements]
https://xxchan.github.io/life/2021/02/11/boox-note-air.html 

 Just some random thoughts of mine.

**Comment by izackwu** [comment]
补充一篇少数派上介绍电子阅读器的文章，很详尽：

https://sspai.com/post/65051

**Reply by Endle** [reply]
墨水大屏看漫画，想想就觉得特别爽
